sysl2/sysl/swagger/tests/SWAGGER_WITH_ENUMS.yaml

```diff
@@ -0,0 +1,71 @@
+swagger: '2.0'
+info:
+  description: foo
+  title: foo
+  version: 0.0.0
+schemes:
+  - https
+
+paths: {}
+
+definitions:
+
+  Attachment:
+    additionalProperties: false
+    properties:
+      data:
+        maxLength: 10000
+        type: string
+      type:
```

Comment:
Nice to have alias for this as well.
```python
type <: AttachementType
```

```python
!alias AttachementType:
        string
```

Reply:
yeah, I'm having a quick look and I'm not sure how to add that in.

I think the way I'm handling alias/array types needs to be reworked to support this.
This ties into my thinking also that `findReferencedType()` is bad.

It really depends on how much we expect to want to extend this to decide if its worth fixing. I think it would be good to dump the enum values as a doc string for the type until sysl gets actual enum support. either way, defer this.